libraries/AP_RangeFinder/AP_RangeFinder_LightWareSerial.cpp

```diff
@@ -21,6 +21,7 @@
 extern const AP_HAL::HAL& hal;
 
 #define LIGHTWARE_DIST_MAX_CM           10000
+#define LIGHTWARE_LOST_SIGNAL_ADD_CM    1000
```

Comment:
10m seems large to me, if user max RNGFND1_MAX_CM close to the lost signal value
why not use LIGHTWARE_OUT_OF_RANGE_ADD_CM

Reply:
This definition has been removed now.